src/lib/components/ScrollView/README.mdx

```diff
@@ -1,7 +1,7 @@
 ---
 name: ScrollView
-menu: 'UI'
-route: /components/ui/scroll-view
+menu: 'Layouts'
```

Comment:
Can you please move `ScrollView` into the misc. category? I've been struggling with finding the right category for it in my today's proposal, but since we are introducing misc., it's better to have it there.

Reply:
done